evacuees/src/UI/embc-evacuee/src/app/sharedModules/components/needs-assessment/needs-assessment-mapping.service.ts

```diff
@@ -153,10 +165,11 @@ export class NeedsAssessmentMappingService {
     }
 
     public convertHouseholdMembers(householdMembers: Array<HouseholdMember>): Array<PersonDetails> {
+        const pathname = window.location.pathname;
```

Comment:
Don't use location function. The different path verified/non-verified can be passed to the service function from the component since it available as route data. Refer review.component for example